From 9f5f0d0d3d1994951c2ac86f0ff492c58cf394e8 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Tue, 14 Apr 2020 15:05:56 +0700
Subject: [PATCH 301/458] arm64: dts: renesas: update threshold temperature of
 system shutdown

Base on HW Manual of RZ/G Series, 2nd Generation, maximum temperature
of normal opreration is 110 degree Celcius.
Therefore, this commit will update this threshold temperature for all
RZ/G2 SoCs.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774a1.dtsi | 6 +++---
 arch/arm64/boot/dts/renesas/r8a774b1.dtsi | 6 +++---
 arch/arm64/boot/dts/renesas/r8a774c0.dtsi | 2 +-
 arch/arm64/boot/dts/renesas/r8a774e1.dtsi | 6 +++---
 4 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a774a1.dtsi b/arch/arm64/boot/dts/renesas/r8a774a1.dtsi
index e8508be..4c2b2a8 100644
--- a/arch/arm64/boot/dts/renesas/r8a774a1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a774a1.dtsi
@@ -2938,7 +2938,7 @@
 
 			trips {
 				sensor1_crit: sensor1-crit {
-					temperature = <120000>;
+					temperature = <110000>;
 					hysteresis = <1000>;
 					type = "critical";
 				};
@@ -2953,7 +2953,7 @@
 
 			trips {
 				sensor2_crit: sensor2-crit {
-					temperature = <120000>;
+					temperature = <110000>;
 					hysteresis = <1000>;
 					type = "critical";
 				};
@@ -2980,7 +2980,7 @@
 				};
 
 				sensor3_crit: sensor3-crit {
-					temperature = <120000>;
+					temperature = <110000>;
 					hysteresis = <1000>;
 					type = "critical";
 				};
diff --git a/arch/arm64/boot/dts/renesas/r8a774b1.dtsi b/arch/arm64/boot/dts/renesas/r8a774b1.dtsi
index dbe4168..54c4636 100644
--- a/arch/arm64/boot/dts/renesas/r8a774b1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a774b1.dtsi
@@ -2737,7 +2737,7 @@
 
 			trips {
 				sensor1_crit: sensor1-crit {
-					temperature = <120000>;
+					temperature = <110000>;
 					hysteresis = <1000>;
 					type = "critical";
 				};
@@ -2752,7 +2752,7 @@
 
 			trips {
 				sensor2_crit: sensor2-crit {
-					temperature = <120000>;
+					temperature = <110000>;
 					hysteresis = <1000>;
 					type = "critical";
 				};
@@ -2787,7 +2787,7 @@
 				};
 
 				sensor3_crit: sensor3-crit {
-					temperature = <120000>;
+					temperature = <110000>;
 					hysteresis = <1000>;
 					type = "critical";
 				};
diff --git a/arch/arm64/boot/dts/renesas/r8a774c0.dtsi b/arch/arm64/boot/dts/renesas/r8a774c0.dtsi
index 9618679..c95f931 100644
--- a/arch/arm64/boot/dts/renesas/r8a774c0.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a774c0.dtsi
@@ -2100,7 +2100,7 @@
 
 			trips {
 				sensor1_crit: sensor1-crit {
-					temperature = <120000>;
+					temperature = <110000>;
 					hysteresis = <2000>;
 					type = "critical";
 				};
diff --git a/arch/arm64/boot/dts/renesas/r8a774e1.dtsi b/arch/arm64/boot/dts/renesas/r8a774e1.dtsi
index b3c85db..3ebfda5 100644
--- a/arch/arm64/boot/dts/renesas/r8a774e1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a774e1.dtsi
@@ -2607,7 +2607,7 @@
 
 			trips {
 				sensor1_crit: sensor1-crit {
-				temperature = <120000>;
+				temperature = <110000>;
 				hysteresis = <2000>;
 				type = "critical";
 				};
@@ -2621,7 +2621,7 @@
 
 			trips {
 				sensor2_crit: sensor2-crit {
-					temperature = <120000>;
+					temperature = <110000>;
 					hysteresis = <2000>;
 					type = "critical";
 				};
@@ -2648,7 +2648,7 @@
 				};
 
 				sensor3_crit: sensor3-crit {
-					temperature = <120000>;
+					temperature = <110000>;
 					hysteresis = <2000>;
 					type = "critical";
 				};
-- 
2.7.4

